### PR TITLE
Launcher/manager complete redesign + some new settings

### PR DESCRIPTION
**Pull request by snouz**
### Mod Manager
* Integrated into launcher (Mods tab)
* Design overhaul (w/ @SamsamTS)
* Mod checkboxes resemble FFIX abilities.

### Launcher
* Full redesign (w/ @SamsamTS)
* 4 menu tabs: Settings, Cheats, Advanced (previously a second window) and Mods
* Settings reorganized by subcategories
* Tooltip overhaul, made to look like FFIX help boxes (with cursor)
* New texts, tooltips, translations and illustration images
  * Spanish fixed by Carú
  * Brazilian-Portuguese fixed by dkPsycho
  * Chinese (Simplified) fixed by imcloud
* Added preset choice (@SamsamTS) with memoria default, steam default and PSX default)
* Added button to save current config as preset, and one to remove presets
* Font list populated on game launching (@SamsamTS)
* New button to copy the content of Memoria.log
* New options:
  * Worldmap: mist presets
  * Worldmap: view distance presets
  * Worldmap: FOV slider
  * Worldmap: Cam height
  * Worldmap speed multiplier
  * Worldmap: camera boost toggle
  * Worldmap: camera tilt toggle
  * Cheat: Tetra Master Easy Win
  * Cheat: Easy jump rope
  * Cheat: Excalibur II no time limit
  * Cheat: Vivi auto magic attack
  * Cheat: Hippaul race
  * Cheat: Swordfight
  * Cheat: Frog catching
  * Advanced: new audio engine
  * Advanced: 3 texture filtering
  * Advanced: Launcher language
  * Advanced: save file option (e.g. disable autosave)
  * 3 Volume control
* Options changed:
  * Shader options simplified into 2 preset boxes
  * Made "TetraMaster reduce randomness" into an option in card mode list
  * Removed Anti-aliasing for now (because it's the culprit of several issues)
  * UI lines x1.5 by default
  * Skip intros ON by default
  * Simplified speed cheat
  * Skip battle intro only on when time is 0
  * Unchecking PSX font sets default pc font
  * Speed cheat 1x = off

### Patcher
* Made more verbose and with colors
* Checks if 100% processed, more error catching
* Closes automatically after 5s if success

### Other
* Fixed some ini functions
* Some package updates

### CODE REVIEW

File: Memoria.Launcher/Languages/zh-CN.xml [hunk 1]
```diff
@@ -161,6 +156,50 @@
...
+	ShaderDropboxChoice4 = "自然"
+	menuSettings = "设置"
+	menuAdvanced = "高级"
+	menuCheats = "秘籍"
+	menuPresets = "预设"
+	Worldmap = "世界地图"
+	QoL = "QoL"
+	Battle = "战斗"
+	VanillaCheats = "基础作弊"
+	EasyTetraMaster = "TetraMaster: 轻松获胜"
```
Comment (imcloud) [inline Memoria.Launcher/Languages/zh-CN.xml:181]: The recommended translation for 'TetraMaster' is' 四方迷雾 ', which is not an official translation but comes from a Chinese mod

Comment (snouz) [reply Memoria.Launcher/Languages/zh-CN.xml:181]: Thanks, added
If you have other adjustments, don't hesitate. As you can guess, I don't speak 10 languages, and these are automatic translations. I'm relying on people to find errors in their native languages.

Comment (imcloud) [reply Memoria.Launcher/Languages/zh-CN.xml]: This Chinese translation file was submitted by me, so I'm tracking updates to the English file and I'd be happy to help

**Comment by snouz** [comment]
Hi @imcloud! We've heavily updated the text of the launcher, would you be ok to proofread them and make sure they don't overflow in the UI?
[Memoria.Launcher.zip](https://github.com/user-attachments/files/17333736/Memoria.Launcher.zip)

**Comment by imcloud** [comment]
> Hi @imcloud! We've heavily updated the text of the launcher, would you be ok to proofread them and make sure they don't overflow in the UI? [Memoria.Launcher.zip](https://github.com/user-attachments/files/17333736/Memoria.Launcher.zip)

ok i will test it as soon as possible

**Comment by imcloud** [comment]
@snouz No overflow found，But what should I do if I want to refine the text? Do I have to PR after your PR? I found some text errors and unfinished sections.

**Comment by snouz** [comment]
> @snouz No overflow found，But what should I do if I want to refine the text? Do I have to PR after your PR? I found some text errors and unfinished sections.

If it's easier for you, you can just send a corrected XML and I'll incorporate it myself!

**Comment by imcloud** [comment]
[zh-CN_2024.10.13.zip](https://github.com/user-attachments/files/17353171/zh-CN_2024.10.13.zip)

@snouz Translated based on the latest commit. "Auto-organize mod by priority" not found.

**Comment by imcloud** [comment]
[zh-CN_2024.10.13.zip](https://github.com/user-attachments/files/17353185/zh-CN_2024.10.13.zip)

@snouz Translated based on the latest commit.

**Comment by snouz** [comment]
> [zh-CN_2024.10.13.zip](https://github.com/user-attachments/files/17353185/zh-CN_2024.10.13.zip)
> 
> @snouz Translated based on the latest commit.

Thank you! That's integrated now

**Pull request merged**